pyomo/solvers/plugins/solvers/CBCplugin.py

```diff
@@ -361,8 +361,11 @@ def process_logfile(self):
                 soln.objective['__default_objective__']['Value'] = float(tokens[2])
             if len(tokens) > 4 and tokens[0] == "Optimal" and tokens[2] == "objective":
                 # parser for log file generetated without discrete variable
-                soln.status = SolutionStatus.optimal
-                soln.objective['__default_objective__']['Value'] = float(tokens[4])
+                try:
```

Comment:
Is this try block needed or is it a remnant of debugging. If it's needed, can you add a bit of documentation about it, try to make the except statement more specific, and remove the print statement.

Reply:
Good catch, it is not needed.  I have removed that line.